src/helpers/abi/ERC20.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
You can use TestToken.json, or replace TestToken.json with ERC20.json because TestToken is also a ERC20 abi.

Reply:
Where you get this abi by the way? Is it from OpenZeppelin?